modules/snowflake-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/snowflake/dsl/SnowflakeFolderMonitoringDao.scala

```diff
@@ -0,0 +1,42 @@
+package com.snowplowanalytics.snowplow.loader.snowflake.dsl
+
+import cats.MonadThrow
+import cats.implicits._
+
+import com.snowplowanalytics.snowplow.rdbloader.common.S3.Folder
+import com.snowplowanalytics.snowplow.rdbloader.algebras.db.FolderMonitoringDao
+import com.snowplowanalytics.snowplow.rdbloader.LoaderError
+import com.snowplowanalytics.snowplow.loader.snowflake.db.{SfDao, Statement, SnowflakeManifest}
+import com.snowplowanalytics.snowplow.loader.snowflake.config.SnowflakeTarget
+
+class SnowflakeFolderMonitoringDao[C[_]: SfDao: MonadThrow](target: SnowflakeTarget) extends FolderMonitoringDao[C] {
```

Comment:
Not necessary. Since there isn't much of code reuse, I would just put raw SQL inside this class, as opposed to moving it to `Statement`.